sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert str((year/week).n(2)) == '52.'
```

Comment:
Ok, that's fine, but I don't think the conversion to a string is a good way to check this. You can compare two floats like `assert abs(answer - expected) < tol` (for comparisons to non-zero numbers).